### ISSUE TITLE
Add resources page should show working

### ISSUE THREAD

**Issue by ahelsing** [GENI-NSF/geni-portal#246]
When you go to create resources, the page just sits there, waiting on the AM.

We should show some kind of feedback. E.G. 'working' or 'waiting for <AM> to stand up your resources' or something.

 _Imported from trac ticket [#246](http://trac.gpolab.bbn.com/proto-ch/ticket/246),  created by ahelsing on 09-27-2012 at 14:31, last modified: 10-30-2012 at 15:34_

**Comment by ahelsing** [comment]
I added a pop-up alert that says 'please be patient'. 

_Trac comment by ahelsing on 10-04-2012 at 09:23_

**Comment by ahelsing** [comment]
this is a duplicate of #250

_Trac comment by ahelsing on 10-30-2012 at 15:34_
